vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -2,570 +2,237 @@
+
+At every phase of the process, options are available to fine tune the process and to see more details, go to the [ent bundle CLI](../getting-started/ent-bundle.md) documentation. 
+
+## Bundle Descriptor entando.json
+The following is a list of specifications in the bundle descriptor and its component parts.
+
+#### Bundle Descriptor Specifications
+|Name|Type|Validation|Description|
+|:-|:-|:-|:-----------------------|
+|`name`|String|Mandatory|Bundle project name. Also used as the default Docker image name|
+|`version`|String|Madatory|Bundle version. It will be used as the default Docker image tag|
+|`displayName`|String||A descriptive label to be used in the UI in place of name|
+|`global`|Global|Optional|Global bundle configuration item|
+|`global: nav`|MenuEntry[]|Optional|Bundle menu global links|
+|`microservices`|Microservices|Optional|Bundle Microservices|
+|`microfrontends`|Micro Frontends|Optional|Bundle Micro Frontends|
+
+```json
+{
+  "name": "bundle-name",
+  "description": "bundle description",
+  "type" : "bundle",
+  "version": "0.0.1",
+  "global": {
+    "nav": [
+      {
+        "label": {
+          "it": "Italian Label",
+          "en": "English Label"
+        },
+        "target": "global-nav-target",
+        "url": "/global-nav-url"
+      }
+    ]
+  }
+}
+```
+
+#### Microservices Specifications
+|Name|Type|Validation|Possible Values|Description|
+|:-|:-|:-|:-|:------------------------|
+|`name`|String|Mandatory||Microservice name|
+|`stack`|Enum|Mandatory|*spring-boot   *node|Microservice stack |
+|`dbms`|Enum|Optional|*none  *embedded  *postgresql  *mysql  *oracle|DBMS required by the MS to provide services|
+|`ingressPath`|String|Optional||Custom ingress path|
+|`healthCheckPath`|String|Optional||Endpoint for a health check|
+|`deploymentBaseName`|String|Optional||Used for defining custom pod names|
+|`roles`|String[]|Optional||Exposed security roles|
+|`env`|EnvironmentVariable[]|Optional||Required environment variables|
+|`commands`|Command[]|Optional||Custom commands definitions|
+
+**Microservices Sample Code**
+```json
+"microservices": [
+    {
+      "name": "ms-name",
+      "stack": "spring-boot",
+      "dbms": "mysql",
+      "ingressPath": "/ingress",
+      "healthCheckPath": "/management/health",
+      "roles": ["admin"],
+      "env": [{ "VAR_1": "value" }]
+    }
+  ],
+```
+
+#### Micro Frontends Specifications
+|Name|Type|Validation|Possible Values|Description|
+|:-|:-|:-|:-|:------------------------|
+|`name`|String|M||Micro frontend name|
+|`stack`|Enum|M|*react   *angular|MFE stack|
+|`type`|Enum|M|*widget  *widget-config  *app-builder|Type of MFE|
+|`slot`|Enum|Mandatory for `type=app-builder`|*primary-header  *primary-menu  *content|Named reference to an App Builder embedded position in a specific layout|
+|`paths`|string[]|Mandatory for `type=app-builder` and `slot=content`||App Builder activation paths|
```

Comment:
I think we need to standardize the Type lists here. It's a little inconsistent in notion. I think String, Enum, String[] or a reference to a lower section (like Command[]) should be the basic pattern. Things like [lang:string] just seem confusing to me.

Reply:
so do I change [lang:string]: string to string[]?

Reply:
String[] seems more consistent. Most of theirs are capitalized.